build.py

```diff
@@ -170,108 +137,130 @@ def get_sources(task_name):
 def main(argv):
     args = getopts(argv)
 
+    used_cores = cor_cores(args.corunner) + [args.core]
+    args.corunner.sort(key=itemgetter(0))
```

Comment:
The first line is a list of the used cores (needed to get the stack objects id). If I remember well, th task core needs to be added to it because a dummy corunner_stack is created for this core when compiling (cleaned by the ld script).

The second line ensures that the corunner list is sorted according to the core. It avoids potential side effects if the script is run two times with a different corunner order.